app/src/main/res/layout/home_disable_exposure_api.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="utf-8"?>
+<FrameLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:clipChildren="false"
+    android:clipToPadding="false"
+    android:paddingTop="65dp"
+    android:paddingBottom="60dp">
```

Comment:
@ioane-bendingspoons It is needed start/end padding to avoid the button to overflow.